Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -177,7 +177,10 @@ internal final class RootViewModel: RootViewModelType, RootViewModelInputs, Root
       .takePairWhen(selectedTabAgain)
       .map { vcs, idx in vcs[idx] }
 
-    self.tabBarItemsData = Signal.combineLatest(currentUser, self.viewDidLoadProperty.signal)
+    self.tabBarItemsData = Signal.combineLatest(currentUser, .merge(
```

Comment:
So now that we want this signal to emit when either of these events occur, we're essentially saying "emit the tab bar items, based on the current user when the view loads or when the current language changes". I find in these situations, `takeWhen` sometimes reads better and _can_ be less error prone than `combineLatest`:
```
self.tabBarItemsData = currentUser
  .takeWhen(.merge(
    self.viewDidLoadProperty.signal,
    self.currentLanguageProperty.signal)
  )
  .map(tabData(forUser:))
```

Reply:
Ah, scratch that - it's not really applicable here because we _also_ want the `currentUser` changing to cause this to emit. I've left the comment here just for consideration should you want to use `takeWhen` in other places.